deno.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  "tasks": {
+    "test": "deno test src && deno fmt --check src && deno lint src",
```

Comment:
Love this, thanks for adding it. Can we also add a quick section to the README on how to run the tests?